lib/generators/manageiq/provider/templates/app/models/%plugin_path%/inventory/persister.rb

```diff
@@ -1,6 +1,4 @@
 class <%= class_name %>::Inventory::Persister < ManageIQ::Providers::Inventory::Persister
-  require_nested :<%= manager_type %>
```

Comment:
@agrare can you take a look at these provider generator changes?  As long as we're autoloading via the conventions, none of these should be required.